spec/models/filters/filter_base_spec.rb

```diff
@@ -148,4 +148,51 @@
       expect(base.on).to eq(today)
     end
   end
+
+  describe 'disabilities filtering' do
+    let(:filter) { Filters::FilterBase.new(user_id: user.id) }
+    let(:hiv_aids_id) { 8 }
+    let(:physical_disability_id) { 5 }
+
+    context 'when user has HIV/AIDS viewing permission' do
+      before do
+        allow(user).to receive(:can_view_hiv_status?).and_return(true)
+      end
```

Comment:
the test double doesn't seem to work for me when I try to run this. Adding the permission seems to work
```suggestion
    let!(:hiv_status_viewer_role) { create :role, can_view_hiv_status: true }

    context 'when user has HIV/AIDS viewing permission' do
      before do
        setup_access_control(user, hiv_status_viewer_role, ds_entity_group)
      end
```